tests/rustworkx_tests/graph/test_all_simple_paths.py

```diff
@@ -243,3 +243,32 @@ def test_all_simple_path_no_path(self):
 
     def test_all_simple_paths_empty(self):
         self.assertEqual({}, rustworkx.all_pairs_all_simple_paths(rustworkx.PyGraph()))
+
+
+class TestGraphLongestSimplePath(unittest.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.graph = rustworkx.generators.cycle_graph(4)
+
+    def test_all_simple_paths(self):
+        res = rustworkx.longest_simple_path(self.graph)
```

Comment:
This seems like a very simple test, as each `n`-node path is a longest path, but since the code is already tested in the DiGraph case I would be fine with this 😄